fix: Multi-cluster installation with revision istiod vs (#14571)
istio/istio.io: content/en/docs/setup/install/multicluster/primary-remote/index.md

    @samples/multicluster/expose-istiod.yaml@
{{< /text >}}

{{< warning >}}

> If the control-plane was installed with a revision `rev`, run

Reviewer: ```suggestion
If the control-plane was installed with a revision `rev`, use the following command instead:
```

{{< text bash >}}

> $ sed 's/{{.Revision}}/rev/g' @samples/multicluster/expose-istiod-rev.yaml.tmpl | kubectl apply -n istio-system -f -

Reviewer: ```suggestion
$ sed 's/{{.Revision}}/rev/g' @samples/multicluster/expose-istiod-rev.yaml.tmpl | kubectl apply --context="${CTX_CLUSTER1}" -n istio-system -f -
```

> $ sed 's/{{.Revision}}/rev/g' @samples/multicluster/expose-istiod-rev.yaml.tmpl | kubectl apply --context="${CTX_CLUSTER1}" -n istio-system -f -

Reviewer: ```suggestion
$ sed 's/{{.Revision}}/rev/g' @samples/multicluster/expose-istiod-rev.yaml.tmpl@ | kubectl apply --context="${CTX_CLUSTER1}" -n istio-system -f -
```
